frontend/src/bundles/studio/components/video-menu/components/mock/menu-mock.tsx

```diff
@@ -1,8 +1,27 @@
+import { Flex, Icon } from '~/bundles/common/components/components.js';
+import { IconName } from '~/bundles/common/icons/icons.js';
+
 const TemplatesContent: React.FC = () => (
     <div>This is the Templates content.</div>
 );
 const AvatarsContent: React.FC = () => <div>This is the Avatars content.</div>;
+const ScriptHeader: React.FC = () => (
+    <Flex justifyContent={'space-between'} w={'280px'}>
+        <div>Script</div>
+        <div>
+            <Icon as={IconName.UPLOAD} />
+        </div>
+    </Flex>
+);
+const ScriptContent: React.FC = () => <div>This is the Script content.</div>;
```

Comment:
We don't need this mocked data for scripts, it is not used in the project anymore

Reply:
Please delete it